AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -145,6 +145,7 @@ whats-on-netflix.com#$#html.sp-message-open body { overflow-y: visible !importan
 !
 !
 !
+morenutrition.de###usercentrics-button
```

Comment:
Where do you need this?

Reply:
Also not seeing it visible on the page

Reply:
Are you watching with the desktop application or with the extension?

Reply:
Android and Windows Chrome Extension.

Reply:
Same. Elemhide rule is not required with the second rule.